servicetalk-opentelemetry/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2018-2019, 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  api platform(project(":servicetalk-dependencies"))
+  testImplementation enforcedPlatform(project(":servicetalk-dependencies"))
+  testImplementation enforcedPlatform("org.junit:junit-bom:$junit5Version")
+  testRuntimeOnly enforcedPlatform(project(":servicetalk-dependencies"))
+
+  api project(":servicetalk-http-api")
+  api "io.opentelemetry:opentelemetry-api"
+
+  implementation project(":servicetalk-annotations")
+  implementation project(":servicetalk-http-utils")
+  implementation "com.google.code.findbugs:jsr305"
+  implementation "org.slf4j:slf4j-api"
+  implementation "io.opentelemetry:opentelemetry-sdk"
+  implementation "io.opentelemetry:opentelemetry-sdk-trace"
```

Comment:
I don't see any imports from these 2 modules, are they required?

Reply:
oh true, they are not required. I removed those.